src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Todo lists</title>
+</head>
+<body>
+    <h1>To Do list</h1>
+    <div id="task_container">
+      <div>
+        <div class="input">
+          <h2 onchange="">Today's To Do</h2>
+          <img
+            src="https://img.icons8.com/windows/32/000000/synchronize.png"
+            alt="sync_button"
+          />
+        </div>
+        <div class="input2">
+          <form class="input"  id="form_Text" action="#">
+            <input
+              type="text"
+              id="inputText"
+              placeholder="Add to your list..."
+            >
+            <button type="button" id="addButton"></button>
+          </form>
+        </div> 
+        <div id="todo_container"></div>
+        <div id="clear_complete">
+          <p>Clear all completed</p>
```

Comment:
While using the p tag for this will not affect the functionality of the clear all completed button I think using the anchor tag or button tag would have been more appropriate.